CHANGELOG.md

```diff
@@ -1,9 +1,8 @@
+<!-- This file is autogenerated. See CONTRIBUTING.md for instructions to add an entry. -->
```

Comment:
https://github.com/open-telemetry/semantic-conventions/blob/482cb01c4598429c65dd0b123b30259f41b360cb/.github/PULL_REQUEST_TEMPLATE.md#L12
needs to be updated accordingly

Reply:
I change it a bit, let me know what you think!